packages/legend-extension-dsl-serializer/src/models/metamodels/pure/model/packageableElements/store/ModelUnit.ts

```diff
@@ -23,18 +23,15 @@ export class ModelUnit implements Hashable {
   packageableElementIncludes: string[];
   packageableElementExcludes: string[];
 
-  constructor(
-    packageableElementIncludes: string[],
-    packageableElementExcludes: string[],
-  ) {
+  constructor() {
     makeObservable(this, {
       packageableElementIncludes: observable,
       packageableElementExcludes: observable,
       hashCode: computed,
     });
 
-    this.packageableElementIncludes = packageableElementIncludes;
-    this.packageableElementExcludes = packageableElementExcludes;
+    this.packageableElementIncludes = [];
```

Comment:
Let's follow the convention and do when you specify the field - after rebasing, you don't need this change due to #557

Reply:
reverted those changes